src/Pandell.PackageRunner/Token.cs

```diff
@@ -0,0 +1,10 @@
+/*
+* IMPORTANT: This file is modified during build process. Don't modify it unless build script is changed as well.
```

Comment:
Does this file get modified during local builds? If so, we might want to add it to `.gitignore` (but still commit it to the repository).